.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Upload Python Package
+
+# on:
+#   release:
+#     types: [created]
+
+on:
+  push: 
+    branches:
+    - dciborow-patch-1
+    - release/*
+    
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+        python setup.py sdist bdist_wheel
+
+    - name: Publish package
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        user: __token__
```

Comment:
this supports a repository url option, can we manually trigger an action and provide a repository url for testpypi?

Reply:
can only trigger manual actions when something is in main branch.